### ISSUE TITLE
false positive for Pandas rule `PD008` by using JAX `.at` syntax

### ISSUE THREAD

**Issue by eringrant** [astral-sh/ruff#8045]
The following `jax_not_pandas.py` script:
```python
from jax import numpy as jnp

x = jnp.empty((100))
x.at[:10].set(1)
```
gives
```sh
$ python -m ruff check --select PD008 --isolated jax_not_pandas.py
jax_not_pandas.py:4:1: PD008 Use `.loc` instead of `.at`. If speed is important, use NumPy.
Found 1 error.
```
even though Pandas is not in use.

**Comment by MeGaGiGaGon** [comment]
This was fixed in #14671, all `PD` rules now make sure pandas is in use before triggering. https://play.ruff.rs/8a4ef5c9-5e26-4995-855f-2bb75f3ec717
